scripts/compile-connectors.ts

```diff
@@ -13,7 +13,7 @@ import chokidar from 'chokidar';
 function generateConnectors() {
 	return new Promise<void>((resolve, reject) => {
 		exec(
-			'tsc --project tsconfig.connectors.json',
+			'esbuild src/connectors/*.ts --bundle --outdir=build/connectorraw',
```

Comment:
Can't we still use the `tsconfig` with esbuild?

See: https://esbuild.github.io/api/#tsconfig

Reply:
Yeah this is a good call. We still have to specify entrypoint though, as esbuild does not care about what typescript says should be included, it takes whatever entrypoints you give it and transforms from there bringing whatever imports it encounters with it.